.env.example

```
@@ -22,6 +22,11 @@ ORDER_DB_URL=jdbc:postgresql://localhost:5432/nextmall
 ORDER_DB_USERNAME=nextmall
 ORDER_DB_PASSWORD=CHANGE_ME
 
+# Database - Checkout Service
+CHECKOUT_DB_URL=jdbc:postgresql://localhost:5432/nextmall
+CHECKOUT_DB_USERNAME=nextmall
+CHECKOUT_DB_PASSWORD=CHANGE_ME
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHECKOUT_DB_PASSWORD key should go before the CHECKOUT_DB_URL key

(UnorderedKey)